chatterbox_tts.py

```
@@ -0,0 +1,201 @@
+                    str(voice_path),
+                    request.temperature,
+                    request.top_p,
+                    request.top_k,
+                    request.repetition_penalty,
+                    request.norm_loudness,
+                )
+                return StreamingResponse(
+                    io.BytesIO(audio_bytes),
+                    media_type="audio/wav",
+                )
+            except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)